src/main/java/landmaster/plustic/modifiers/armor/JetpackPancakeHippos.java

```diff
@@ -110,14 +119,10 @@ public String getStateMsgUnloc() {
 	
 	private static final Map<KeyBinding, JetpackSettings> keys = new HashMap<>();
 	static {
-		try {
```

Comment:
A try catch around a map insert does not stop you from inserting null into said map. SimplyJetPacks KeybindHandler gets initialized in init but this gets called in pre init.